pages/AmazonLoginPage.js

```diff
@@ -0,0 +1,45 @@
+
+module.exports =class AmazonLoginPage {
+   constructor(page){
+     this.page = page
+   }
+
+   async open(){
+      await this.page.goto('https://www.amazon.in/ap/signin?ie=UTF8&openid.pape.max_auth_age=0&openid.return_to=https%3A%2F%2Fwww.amazon.in%2Fyour-account%3Fref_%3Dnav_signin&openid.identity=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.assoc_handle=inflex&openid.mode=checkid_setup&openid.claimed_id=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&switch_account=signin&ignoreAuthState=1&disableLoginPrepopulate=1&ref_=ap_sw_aa', {
```

Comment:
why you are using such a long URL? Add this URL to `.env` file

Reply:
Done